armi/physics/neutronics/globalFlux/globalFluxInterface.py

```diff
@@ -836,6 +836,18 @@ def updateFluenceAndDpa(self, stepTimeInSeconds, blockList=None):
             b.p.detailedDpaPeak = b.p.detailedDpaPeak + newDPAPeak
             b.p.detailedDpaThisCycle = b.p.detailedDpaThisCycle + newDpaThisStep
 
+            # increment point dpas
+            # this is specific to hex geometry, but they are general neutronics block parameters
+            # if it is a non-hex block, this should be a no-op
+            if b.p.pointsCornerDpa is not None and b.p.pointsCornerDpaRate is not None:
```

Comment:
`pointsCornerDpa` and `pointsEdgeDpa` is `None` out of the gate. If we're moving this code from the internal plugin, how are these variables getting initialized to empty numpy arrays (since that plugin was doing that work)?

Reply:
Yes, they need to be initialized